src/concrete_minkowski_difference.jl

```diff
@@ -0,0 +1,40 @@
+export minkowski_difference, pontryagin_difference
+
+"""
+    minkowski_difference(Q::LazySet{N}, P::LazySet{N}) where {N<:Real}
```

Comment:
The docs fail because the new function's docstring should be added to the docs (the docs build is strictin that sense, they fail if some docstring is missing).

You can add a new subsection `## Minkowski difference` in the file `binary_functions.md`, https://github.com/JuliaReach/LazySets.jl/blob/master/docs/src/lib/binary_functions.md#minkowski-sum   like so:

```julia
minkowski_difference(::LazySet{N}, ::LazySet{N}) where {N<:Real}
pontryagin_difference
```

Reply:
Do I need to add the arguments also for the function pontryagin_difference, or is this omitted because it is an alias?

Reply:
Correct; you can ommit the args because there is only 1 method `pontryagin_difference`.